modules/reset_codebuild.tf

```diff
@@ -108,14 +108,8 @@ resource "aws_codebuild_project" "reset_build" {
     }
 
     environment_variable {
-      name  = "RESET_NAMESPACE"
```

Comment:
`RESET_NAMESPACE` and `IS_PR` env vars aren't being used anywhere (did a full code search to double-check)